### PR TITLE
xmlto: workaround -o argument handling with cygpath

### PR DESCRIPTION
**Pull request by elmarco**
Patch sent upstream for review:
https://pagure.io/xmlto/pull-request/10

### CODE REVIEW

**Comment by elmarco** [comment]
wrong patch, oops

File: xmlto/PKGBUILD [hunk 1]
```diff
@@ -2,16 +2,23 @@
...
 makedepends=('docbook-xsl' 'autotools' 'gcc')
-source=("https://releases.pagure.org/${pkgname}/${pkgname}-${pkgver}.tar.bz2")
-sha256sums=('1130df3a7957eb9f6f0d29e4aa1c75732a7dfb6d639be013859b5c7ec5421276')
+source=("https://releases.pagure.org/${pkgname}/${pkgname}-${pkgver}.tar.bz2"
+        0001-xmlto-convert-output-path-to-unix-path-on-cygwin-msy.patch)
+sha256sums=('1130df3a7957eb9f6f0d29e4aa1c75732a7dfb6d639be013859b5c7ec5421276'
+            '981e200d9470852048df58fa7afbd02264b75d81b8e572c1aba3dc1487ad0f9d')
+
+prepare() {
+  cd "${srcdir}/${_realname}-${pkgver}"
```
Comment (mmuetzel) [inline xmlto/PKGBUILD:19]: Should be `cd "${srcdir}/${pkgname}-${pkgver}"`?

**Comment by lazka** [comment]
How/Why is something passing Windows paths to xmlto?

**Comment by elmarco** [comment]
> How/Why is something passing Windows paths to xmlto?

This happens with meson:
```
custom_target(...
            command: [
                xmlto, 'man', '@INPUT@', '-o', meson.current_build_dir()
            ],
)
```

**Comment by lazka** [comment]
I see. Imho adding hacks to every unix tool to not mangle Windows paths is a bit futile and paths should be converted on the non-cygwin side of things.

But this seems easy/manageable enough, so fine with me.

**Comment by elmarco** [comment]
> I see. Imho adding hacks to every unix tool to not mangle Windows paths is a bit futile and paths should be converted on the non-cygwin side of things.
> 
> But this seems easy/manageable enough, so fine with me.

So meson shouldn't generate C:/ windows paths in the first place? Based on what environment should they make that decision?

**Comment by lazka** [comment]
The meson build file could use cygpath when run on Windows, but yeah, mixing cygwin and native is always hacky..

**Pull request merged**